src/components/PeopleFilters.tsx

```diff
@@ -33,63 +81,61 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {centuriesParams.map(c => (
+              <Link
+                key={c.sortBy}
+                data-cy="century"
+                className={cn('button mr-1', {
+                  'is-info': centuries.includes(c.sortBy),
+                })}
+                to={{
+                  search: getSearchWith(
+                    {
+                      centuries: centuries.includes(c.sortBy)
+                        ? centuries.filter(cent => cent !== c.sortBy)
+                        : [...centuries, c.sortBy],
+                    },
+                    searchParams,
+                  ),
+                }}
+              >
+                {c.sortBy}
+              </Link>
+            ))}
```

Comment:
Here you are using 'c.sortBy' as a key for your React components. It's good that you are not using the index, but make sure that 'c.sortBy' is unique for each item.